client/pages/Borrow.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import Layout from "../components/Layout";
+
+export default function borrow() {
+  return (
+    <Layout>
+      <div>Borrow</div>
+    </Layout>
+  );
+}
```

Comment:
Let's keep the file name in lowercase (`borrow.js`) and the function name to title case (`Borrow()`) since routes will be looking out for `/borrow` and not `/Borrow`

Reply:
yes agreed